flexvalue/calculations.py

```diff
@@ -682,9 +690,30 @@ def get_results(self, user_inputs):
         )
         elec_benefits = self.get_all_trc_electricity_benefits_df(user_inputs)
         gas_benefits = self.get_total_trc_gas_benefits(user_inputs)
-
         # if index wasn't already set with the ID colum, set it for joining to the output
         if user_inputs.index.name != "ID":
             user_inputs = user_inputs.set_index("ID")
         outputs_table = user_inputs.join(outputs_table).reset_index()
         return outputs_table, outputs_table_totals, elec_benefits, gas_benefits
+
+    def get_time_series_results(self, user_inputs):
```

Comment:
so then the thought is here that a user would call either the get_results method OR the get_time_series_results method right?

Reply:
That is correct yes.  I considered adding them to get_results, and in fact get_results already returns the time series elec_benefits, but for some reason it sums up the gas_benefits to a total.  I would have had to add another output to get_results to include the time series gas outputs and I was worried about breaking existing code in the wild.